api/src/main/java/org/openmrs/module/patientqueueing/QueueingUtils.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.patientqueueing;
+
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class QueueingUtils {
+	
+	private QueueingUtils() {
+	}
+	
+	/**
+	 * This method converts date to date format of set globalproperty
+	 * patientqueueing.defaultDateFormat and adds time incase its set
+	 * 
+	 * @param date This is the date to be converted into set date format
+	 * @param time The time String which is appended to the date in-case it is not null
+	 * @return Date which is converted to required format
+	 * @throws ParseException
+	 */
+	public static Date convertDateToDefaultDateFormat(Date date, String time) throws ParseException {
```

Comment:
Am struggling to understand what you need to convert a date object to another date object.

Reply:
@dkayiwa I have renamed this hope it makes sense